components/expenses/MarkExpenseAsUnpaidBtn.js

```diff
@@ -31,6 +31,7 @@ const MarkExpenseAsUnpaidBtn = ({ id, markExpenseAsUnpaid, refetch }) => {
     } catch (err) {
       console.log('>>> payExpense error: ', err);
       setState({ ...state, disableBtn: false });
+      onError(err);
```

Comment:
Here the error is not wrapped with `getErrorFromGraphqlException`, is it intentional?

Reply:
No that wasn't intentional, I thought I had checked it in.